guitest / run (windows-latest): ERROR: The system cannot find the file specified. libsodium-1.0.17-msvc.zip

```
Run ./.github/actions/windows_dependencies
Run actions/cache@v3
Cache not found for input keys: Windows-cached-bin
Run if not exist "cached-bin" mkdir "cached-bin"
Run andstor/file-existence-action@v2
These files don't exist: cached-bin/libsodium-1.0.17/libsodium.dll
Run C:\msys64\usr\bin\wget.exe -q https://download.libsodium.org/libsodium/releases/libsodium-1.0.17-msvc.zip

7-Zip 22.01 (x64) : Copyright (c) 1999-2022 Igor Pavlov : 2022-07-15


ERROR: The system cannot find the file specified.
libsodium-1.0.17-msvc.zip
Scanning the drive for archives:



System ERROR:
The system cannot find the file specified.
The system cannot find the path specified.
Error: Process completed with exit code 1.
```

https://github.com/Tribler/tribler/actions/runs/5058208197/jobs/9095409415?pr=7434

--- a/.github/actions/windows_dependencies/action.yml
+++ b/.github/actions/windows_dependencies/action.yml
@@ -29,12 +29,17 @@ runs:
 
     - name: Download Libsodium
       if: steps.check_libsodium.outputs.files_exists == 'false'
+      env:
+        RELEASE_FOLDER: .\libsodium\x64\Release
       shell: cmd
       run: |
         C:\msys64\usr\bin\wget.exe -q https://download.libsodium.org/libsodium/releases/libsodium-${{inputs.libsodium-version}}-msvc.zip
         7z x libsodium-${{inputs.libsodium-version}}-msvc.zip
+        dir %RELEASE_FOLDER% /ad /b /o-n > latest_release_file
+        set /p latest_release= < latest_release_file
+        echo Latest release: %latest_release%
         if not exist "cached-bin\libsodium-${{inputs.libsodium-version}}" mkdir "cached-bin\libsodium-${{inputs.libsodium-version}}"
-        copy .\x64\Release\v141\dynamic\libsodium.dll .\cached-bin\libsodium-${{inputs.libsodium-version}}\libsodium.dll
+        copy %RELEASE_FOLDER%\%latest_release%\dynamic\libsodium.dll .\cached-bin\libsodium-${{inputs.libsodium-version}}\libsodium.dll
 
     - name: Install Libsodium
       shell: cmd

--- a/.github/actions/windows_dependencies/action.yml
+++ b/.github/actions/windows_dependencies/action.yml
@@ -2,7 +2,7 @@ name: windows_dependencies
 description: Setup windows dependencies for Tribler
 inputs:
   libsodium-version:
-    default: '1.0.17'
+    default: '1.0.18'
     description: 'Libsodium version'
     required: false
 

